src/platform_impl/ios/view.rs

```diff
@@ -519,7 +519,15 @@ pub unsafe fn create_window(
             msg_send![window, setScreen:video_mode.monitor().ui_screen()]
         }
         Some(Fullscreen::Borderless(ref monitor)) => {
-            msg_send![window, setScreen:monitor.ui_screen()]
+            let uiscreen: id = match monitor {
+                Some(ref monitor) => monitor.ui_screen() as id,
+                None => {
+                    let uiscreen: id = msg_send![window, screen];
+                    uiscreen
```

Comment:
Type inference issues I'd assume?

Reply:
Yeah, it's really strange how it worked, but returning `msg_send!` wasn't working. I'm not sure that there's a CI log for that, but it was somewhere.